src/controllers/account.js

```diff
@@ -25,7 +28,7 @@ class AccountController {
 
             user.password = await bcrypt.hash(newPassword, 10);
 
-            await UserService.updateUser(user);
+            await Promise.all([SessionService.removeAllSessionsByUser(user._id), UserService.updateUser(user)]);
```

Comment:
If some of promises rejected you need to check if and return error message. Also in all controllers use "try catch" for error handling